tools/validators/instance_validator/tests/handler_test.py

```diff
@@ -45,6 +44,17 @@ def testValidateOneBuildingExistFails(self):
       instance_handler = handler.ValidationHelper(args)
```

Comment:
A question about this test (github won't let me add a comment on line 40/39): why is the test file GOOD/good_links.yaml in the GOOD directory if it causes a SyntaxError to be raised?

I noticed that there is an entity commented out in good_links.yaml as well. Is that the reason the error is raised? Can we remove the commented lines and move the file to BAD/bad_links.yaml?

Reply:
+1

Reply:
Cool, will do this in a separate PR.